src/editing.md

```diff
@@ -1,463 +1,489 @@
 
-When the card comes up for review, it will appear as:
+"Lückentext"-Notiztypen werden von Anki speziell behandelt und können deswegen
+nicht auf einem regulären Notiztyp basieren. Um einen solchen Notiztyp anzupassen
+musst du den Standardnotiztyp "Lückentext" Klonen.
+Formatierung und Stil können wie bei regulären Notiztypen angepasst werden,
+es ist jedoch nicht möglich, zusätzliche Kartenvorlagen zu einem Notiztyp
+hinzuzufügen, der auf dem Typ "Lückentext" basiert.
 
-    [city] was founded in 1913.
+## Eingabe von Sonderzeichen und Akzenten
 
-For information on testing your ability to type in a cloze deletion
-correctly, please see the section on [typing answers](templates/fields.md#checking-your-answer).
+Alle modernen Computer unterstützen die Eingabe fremder Symbole und Akzente
+auf unterschiedliche Art und Weise. Wir empfehlen, die Tastaturbelegung der
+Sprache zu verwenden, die du lernen willst.
 
-Please note that overlapping clozes are not supported. For example, the
-following field is invalid:
+Sprachen, die ihre eigene Schrift haben (z. B. Japanisch, Chinesisch, Russisch usw.)
+haben stets auch spezifische Tastaturbelegungen.
 
-    {{c1::Canberra was {{c2::founded}}}} in 1913
+Europäische Sprachen, die Akzente benutzen, haben oftmals auch eigene Belegungen,
+können aber auch mit der deutschen Tastaturbelegung geschrieben werden.
+Hierfür musst du zuerst den gewünschten Akzent und dann den Buchstaben eingeben,
+der den Akzent erhalten soll - z. B. ergeben <kbd>`</kbd> und <kbd>A</kbd> das Symbol à.
 
-If you need to create clozes from overlapping text, add another Text
-field to your cloze, add it to the [template](templates/intro.md), and then when
-creating notes, paste the text into two separate fields, like so:
+<!-- deleted paragraph -->
 
-    Text1 field: {{c1::Canberra was founded}} in 1913
+<!-- deleted paragraph -->
 
-    Text2 field: {{c2::Canberra}} was founded in 1913
+Um herauszufinden, wie du fremde Tastaturbelegungen auf deinem Betriebssystem installieren
+kannst, gib in Google Suchbegriffe wie "Japanische Tastaturbelegung Mac" oder
+"Chinesische Tastaturbelegung Windows" ein.
 
-The default cloze note type has a second field called Extra, that is
-shown on the answer side of each card. It can be used for adding some
-usage notes or extra information.
+Für Sprachen mit umgekehrter Leserichtung (RTL) wird es etwas komplizierter.
+Diese [Website](http://dotancohen.com/howto/rtl_right_to_left.html) (Englisch)
+bietet einen guten Ausgangspunkt.
 
-The cloze note type is treated specially by Anki, and cannot be created
-based on a regular note type. If you wish to customize it, please make
-sure to clone the existing Cloze type instead of another type of note.
-Things like formatting can be customized, but it is not possible to add
-extra card templates to the cloze note type.
+Das Toolkit, auf dem Anki's Editor aufgebaut ist, hat Schwierigkeiten mit manchen
```

Comment:
```suggestion
Das Toolkit, auf dem Ankis Editor aufgebaut ist, hat Schwierigkeiten mit manchen
```

Reply:
No sure about this one. According to Duden, the apostrophe is allowed for brands/company names to emphasise the name: https://www.duden.de/sprachwissen/sprachratgeber/Ehrenrettung-fur-Grossmutters-Apfelkuchen.

I wouldn't say: "Twitters Buchstabenlimit geht mir ordentlich auf den Keks." - but use "Twitter's" instead to improve readability.

Reply:
> According to Duden, the apostrophe is allowed for brands/company names to emphasise the name:

I don't see this rule in the linked source. Instead I read:

> Zur Verdeutlichung der Grundform ist diese Ausnahme von der Regel beim Genitiv möglich – aber ***nur bei Personennamen*** und nur, wenn’s denn unbedingt sein soll.

Reply:
Yeah, sorry I missed that. GitHub collapses all these comments which makes it a bit hard to find them later on.
Let's just use Ankis, no biggie :+1: